### PR TITLE
use https on links which support it

### PR DESCRIPTION
**Pull request by JakeChampion**
This will remove the mixed content warning we get on the origami-image-service homepage.
This will also use https for more of our image backends such as prod-upp-image-read.ft.com and im.ft-static.com 👍

### CODE REVIEW

File: test/integration/v2/images-debug.test.js [hunk 1]
```diff
@@ -61,11 +61,11 @@ describe('GET /v2/images/debug…', function () {
 	const imagesWithSchemes = {
 		ftcms: {
 			requestedUrl: 'ftcms:6c5a2f8c-18ca-4afa-80ff-7d56e41172b1',
-			resolvedUrl: new RegExp('http://im.ft-static.com/content/images/6c5a2f8c-18ca-4afa-80ff-7d56e41172b1.img')
+			resolvedUrl: new RegExp('https://im.ft-static.com/content/images/6c5a2f8c-18ca-4afa-80ff-7d56e41172b1.img')
```
Comment (JakeChampion) [inline test/integration/v2/images-debug.test.js:64]:
```suggestion
			resolvedUrl: new RegExp('https://im\\.ft-static.com/content/images/6c5a2f8c-18ca-4afa-80ff-7d56e41172b1\\.img')
```

File: test/integration/v2/images-debug.test.js [hunk 2]
```diff
@@ -61,11 +61,11 @@ describe('GET /v2/images/debug…', function () {
...
 		},
 		capiv1: {
 			requestedUrl: 'ftcms:be875529-7675-43d8-b461-b304410398c2',
-			resolvedUrl: new RegExp('http://im.ft-static.com/content/images/be875529-7675-43d8-b461-b304410398c2.img')
+			resolvedUrl: new RegExp('https://im.ft-static.com/content/images/be875529-7675-43d8-b461-b304410398c2.img')
```
Comment (JakeChampion) [inline test/integration/v2/images-debug.test.js:68]:
```suggestion
			resolvedUrl: new RegExp('https://im\\.ft-static.com/content/images/be875529-7675-43d8-b461-b304410398c2\\.img')
```

File: test/integration/v2/images-debug.test.js [hunk 3]
```diff
@@ -96,20 +96,20 @@ describe('GET /v2/images/debug…', function () {
 			resolvedUrl: new RegExp('https://d1e00ek4ebabms.cloudfront.net/production/817dd37c-b808-4b32-9db2-d50bdd92372b.jpg')
 		},
 		httpftcms: {
-			requestedUrl: 'http://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img',
-			resolvedUrl: new RegExp('http://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img')
+			requestedUrl: 'https://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img',
+			resolvedUrl: new RegExp('https://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img')
```
Comment (JakeChampion) [inline test/integration/v2/images-debug.test.js:100]:
```suggestion
			resolvedUrl: new RegExp('https://im\\.ft-static\\.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2\\.img')
```

File: test/integration/v2/images-debug.test.js [hunk 4]
```diff
@@ -96,20 +96,20 @@ describe('GET /v2/images/debug…', function () {
...
 		},
 		httpsftcms: {
 			requestedUrl: 'https://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img',
-			resolvedUrl: new RegExp('http://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img')
+			resolvedUrl: new RegExp('https://im.ft-static.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2.img')
```
Comment (JakeChampion) [inline test/integration/v2/images-debug.test.js:104]:
```suggestion
			resolvedUrl: new RegExp('https://im\\.ft-static\\.com/content/images/a60ae24b-b87f-439c-bf1b-6e54946b4cf2\\.img')
```

**Review by notlee — APPROVED** [review]
lovely

**Pull request merged**